lib/disposable.dart

```diff
@@ -37,13 +37,25 @@ class _InternalDisposable implements _Disposable {
   }
 }
 
+/// An interface to abstract management from actual disposal machinery.
```

Comment:
This read a bit weird to me. If anyone else agrees, can we consider tweaking it?